src/components/weather/Weather.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {Dispatch, SetStateAction, useEffect, useState} from 'react';
+import {weatherAPI, WeatherAPIType } from '../../api/weatherAPI';
+import {InformationOfCity} from '../informationOfCity/InformationOfCity';
+
+type PropsType = {
+    coordinates?: string
+}
+
+export const Weather: React.FC<PropsType> = (props) => {
+
+    const [state, setState] = useState<WeatherAPIType>({
```

Comment:
`const [state, setState] = useState<WeatherAPIType>({initialState});`
move initialState to the const

Reply:
understood, but it's much better to move const, in the same file or outside, then where?